ecospheres_migrator/migrator.py

```diff
@@ -38,6 +46,28 @@ class Transformation:
     def name(self) -> str:
         return self.path.stem
 
+    @cached_property
+    def params(self) -> list[TransformationParam]:
+        xslt = etree.parse(self.path, parser=None)
+        root = xslt.getroot()
+        params = []
+        ns = {"xsl": "http://www.w3.org/1999/XSL/Transform"}
+        for param in root.xpath("//xsl:param", namespaces=ns):
```

Comment:
```suggestion
        for param in root.xpath("//xsl:param", namespaces=root.nsmap):
```
I didn't test specifically here but `root.nsmap` contains all namespaces declared in the parased file, so this should be enough.